file_formats/cmt_format.mli

```diff
@@ -44,6 +44,7 @@ and binary_part =
   | Partial_structure of structure
   | Partial_structure_item of structure_item
   | Partial_expression of expression
+  | Partial_expr_poly of expr_poly
```

Comment:
At first sight I don't think we need a "partial" case for a minor construct such as the new one. In fact, the only tool that I know of that uses this information is `cmt2annot` which is pretty much deprecated and unused (who uses `.annot` files nowadays?).

Reply:
Just removing this case and all the corresponding uses seems a better approach ?

Reply:
Yes, I think so.